src/Buffer.h

```diff
@@ -568,12 +568,12 @@ class Buffer {
     Expr operator()(const Expr &first, Args... rest) const {
         std::vector<Expr> args = {first, rest...};
         return (*this)(args);
-    };
+    }
```

Comment:
Ooh, I'd actually love to enable whatever warnings complains about this, it's a minor pet peeve of mine